src/Storage/Controllers/MessageboxInstancesController.cs

```diff
@@ -417,18 +417,18 @@ private async Task RemoveHiddenInstances(List<Instance> instances)
 
             if (queryModel.FromCreated != null)
             {
-                queryParams.Add("created", $"gte:{queryModel.FromCreated?.ToString(dateTimeFormat)}");
+                queryParams.Add("msgBoxInterval", $"gte:{queryModel.FromCreated?.ToString(dateTimeFormat)}");
             }
 
             if (queryModel.ToCreated != null)
             {
-                if (queryParams.TryGetValue("created", out StringValues createdValues))
+                if (queryParams.TryGetValue("msgBoxInterval", out StringValues createdValues))
```

Comment:
Should we try to avoid using the term creatd then except for when referring to the queryModel ? 
```suggestion
                if (queryParams.TryGetValue("msgBoxInterval", out StringValues msgBoxIntervalValues))
```

Reply:
Gjort